### DISCUSSION TITLE
Question on Proxy

### DISCUSSION THREAD

**Discussion by killmasta93** [postalserver/postal#3093] [Help with using Postal] [answered]
Hi
I was wondering if someone could shed some light, Currently i have my nginx proxy on another VM and i installed postal on another VM,
as i need the port 5000 to be able to see each other in the same LAN but i was checking on the ports sections that only lets me on 127.0.0.1

`ruby      3133991             999    5u  IPv4 1114445907      0t0  TCP 127.0.0.1:5000 (LISTEN)
`
as i would need it to be similar like this 
`
 docker-pr     648            root    4u  IPv4      25348      0t0  TCP *:9000 (LISTEN)
`
Not sure what part i should add on the docker compose? or on the config?
Thank you

**Comment by willpower232** [comment, marked as answer]
You probably want to change a bind address in either the yaml or environment variables

https://github.com/postalserver/postal/blob/main/doc/config/yaml.yml
https://github.com/postalserver/postal/blob/main/doc/config/environment-variables.md

**Reply by killmasta93** [reply]
Thank you that did the trick
